lib/processes.js

```diff
@@ -43,6 +46,24 @@ var App = require('./processes/application');
 var Selenium = require('./processes/selenium');
 var ChromeDriver = require('./processes/chromedriver');
 
+var CHROME = 'chrome';
+
+function isChromeInDocker(browserName) {
+  if (browserName !== CHROME) {
+    return Bluebird.reject();
+  }
+
+  var fileName = '/.dockerenv';
```

Comment:
so we could possibly check for other things in addition to `/.dockerenv` but this seemed like it would work for our needs. 

I suppose the other option is that we just check if `process.getuid() === 0` but looking for `.dockerenv` seemed like a better way to make sure.  That said I can't think of a reason why we would ever be running as `0` except in docker at the moment